app/services/moves/params_validator.rb

```diff
@@ -34,6 +36,7 @@ def initialize(filter_params, sort_params)
       @date_of_birth_from = filter_params[:date_of_birth_from]
       @date_of_birth_to = filter_params[:date_of_birth_to]
       @move_type = filter_params[:move_type]
+      @status = filter_params[:status]
```

Comment:
Is the node application going to care about the status soon, is that the plan? It will currently ignore this attribute because it doesn't know about it.

Reply:
interesting question - when I was looking at the validators I noticed that `status` was missing from the list of things to validate and I assumed that was a mistake - but perhaps not if the node app isn't using status. But no harm adding it here in case it is used in future.

Reply:
Hmm, famous last words; adding this validation has broken something; I'll remove it in another PR